### ISSUE TITLE
Lua Error

### ISSUE THREAD

**Issue by Audixx** [Meivyn/AdventureGuideLockouts#7]
Keep getting this error, i've deleted the addon and re-installed, disabled every addon i have and i still get the error

```
Message: ...nalSavedInstances\EncounterJournalSavedInstances.lua:278: table index is nil
Time: Mon Sep 17 18:17:27 2018
Count: 1
Stack: ...nalSavedInstances\EncounterJournalSavedInstances.lua:278: table index is nil
...nalSavedInstances\EncounterJournalSavedInstances.lua:278: in function <...nalSavedInstances\EncounterJournalSavedInstances.lua:233>
...nalSavedInstances\EncounterJournalSavedInstances.lua:533: in function `UpdateFrames'
...nalSavedInstances\EncounterJournalSavedInstances.lua:546: in function <...nalSavedInstances\EncounterJournalSavedInstances.lua:527>

Locals: (for index) = 2
(for limit) = 29
(for step) = 1
i = 2
instanceName = "The Eye of Eternity"
_ = 291898439
reset = 135758
instanceDifficulty = 4
_ = true
_ = false
_ = 524582912
_ = true
_ = 25
difficultyName = "25 Player"
maxBosses = 1
defeatedBosses = 1
instanceLink = "[The Eye of Eternity]"
instanceID = nil
difficulty = "normal"
bosses = <table> {
 1 = <table> {
 }
}
b = 2
encounters = <table> {
 1 = 2168
 2 = 2167
 3 = 2169
 4 = 2146
 5 = 2166
 6 = 2195
 7 = 2194
 8 = 2147
}
(*temporary) = <table> {
 759 = <table> {
 }
}
(*temporary) = <table> {
}
(*temporary) = 2
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = 2147
(*temporary) = "table index is nil"
savedInstances = <table> {
 759 = <table> {
 }
}
RequestRaidInfo = <function> defined =[C]:-1
GetNumSavedInstances = <function> defined =[C]:-1
GetSavedInstanceInfo = <function> defined =[C]:-1
GetSavedInstanceChatLink = <function> defined =[C]:-1
tonumber = <function> defined =[C]:-1
instancesData = <table> {
 575 = 286
 599 = 277
 249 = 760
 615 = 755
 1004 = 316
 631 = 758
 33 = 64
 1501 = 740
 1175 = 385
 1862 = 1021
 544 = 747
 552 = 254
 409 = 741
 568 = 77
 576 = 281
 1771 = 1002
 1176 = 537
 600 = 273
 608 = 283
 1841 = 1022
 624 = 753
 632 = 280
 1594 = 1012
 1877 = 1030
 1822 = 1023
 1864 = 1036
 545 = 261
 469 = 742
 1762 = 1041
 1754 = 1001
 1763 = 968
 585 = 249
 720 = 78
 601 = 272
 1712 = 946
 1861 = 1031
 752 = 67
 509 = 743
 671 = 72
 649 = 757
 657 = 68
 1195 = 558
 546 = 262
 554 = 258
 1648 = 861
 996 = 320
 578 = 282
 967 = 187
 1493 = 707
 602 = 275
 1466 = 762
 1228 = 477
 1007 = 246
 1571 = 800
 1530 = 786
 650 = 284
 531 = 744
 1651 = 860
 547 = 260
 555 = 253
 1753 = 945
 1492 = 727
 960 = 313
 1477 = 721
 595 = 279
 603 = 759
 1456 = 716
 619 = 271
 754 = 74
 1516 = 726
 643 = 65
 1358 = 559
 532 = 745
 540 = 259
 548 = 748
 556 = 252
 564 = 751
 1136 = 369
 580 = 752
 1676 = 875
 1544 = 777
 604 = 274
 1448 = 669
 1001 = 311
 755 = 69
 1205 = 457
 644 = 70
 1677 = 900
 533 = 754
 668 = 276
 1279 = 556
 557 = 250
 565 = 746
 1209 = 476
 962 = 303
 1208 = 536
 724 = 761
 859 = 76
 994 = 321
 1182 = 547
 1098 = 362
 1520 = 768
 645 = 66
 1009 = 330
 269 = 255
 542 = 256
 550 = 749
 558 = 247
 1008 = 317
 574 = 285
 961 = 302
 1011 = 324
 959 = 312
 1458 = 767
 669 = 73
 939 = 185
 757 = 75
 534 = 750
 670 = 71
 938 = 184
 36 = 63
 543 = 248
 658 = 278
 940 = 186
 553 = 257
 560 = 251
}
GetSavedInstanceEncounterInfo = <function> defined =[C]:-1
tremove = <function> defined =[C]:-1
EJ_GetEncounterInfo = <function> defined =[C]:-1
tinsert = <function> defined =[C]:-1
UnitFactionGroup = <function> defined =[C]:-1
worldBossesData = <table> {
 959 = <table> {
 }
 557 = <table> {
 }
 322 = <table> {
 }
 822 = <table> {
 }
 1028 = <table> {
 }
}
GetQuestTimeLeftMinutes = <function> defined =[C]:-1
GetQuestTagInfo = <function> defined =[C]:-1
IsQuestFlaggedCompleted = <function> defined =[C]:-1
pairs = <function> defined =[C]:-1
ipairs = <function> defined =[C]:-1
WORLD_BOSS = "World Boss"
```

**Comment by Meivyn** [comment]
Looks like I missed The Eye of Eternity raid in my ``instancesData`` table. Thanks for pointing it out.

Fixed in 8920bc57bee35f852be66ea2938a0889a087798f.

**Comment by Meivyn** [comment]
Fixed in 8920bc57bee35f852be66ea2938a0889a087798f.
